packages/components/date-picker/src/date-picker-com/panel-date-range.vue

```diff
@@ -481,10 +481,10 @@ export default defineComponent({
 
     const formatEmit = (emitDayjs: Dayjs, index?) => {
       if (!emitDayjs) return
-      if (defaultTime) {
-        const defaultTimeD = dayjs(defaultTime[index] || defaultTime).locale(
-          lang.value
-        )
+      if (defaultTime.value) {
```

Comment:
I think we should try to avoid redundant dependency collection, same for other places

```js
const defaultTimeValue = defaultTime.value
const defaultTimeD = dayjs(defaultTimeValue[index] || defaultTimeValue).locale(lang.value)
```

Reply:
fine